tests/test_ncm_stats_dist_nd.c

```diff
@@ -45,45 +45,62 @@ typedef struct _TestNcmStatsDistNd
 } TestNcmStatsDistNd;
 
 static void test_ncm_stats_dist_nd_new_kde_gauss (TestNcmStatsDistNd *test, gconstpointer pdata);
-static void test_ncm_stats_dist_nd_gauss_dens_est (TestNcmStatsDistNd *test, gconstpointer pdata);
-static void test_ncm_stats_dist_nd_gauss_dens_interp (TestNcmStatsDistNd *test, gconstpointer pdata);
-static void test_ncm_stats_dist_nd_gauss_dens_interp_unormalized (TestNcmStatsDistNd *test, gconstpointer pdata);
-static void test_ncm_stats_dist_nd_gauss_sampling (TestNcmStatsDistNd *test, gconstpointer pdata);
+static void test_ncm_stats_dist_nd_new_kde_studentt (TestNcmStatsDistNd *test, gconstpointer pdata);
+
+static void test_ncm_stats_dist_nd_dens_est (TestNcmStatsDistNd *test, gconstpointer pdata);
+static void test_ncm_stats_dist_nd_dens_interp (TestNcmStatsDistNd *test, gconstpointer pdata);
+static void test_ncm_stats_dist_nd_dens_interp_unormalized (TestNcmStatsDistNd *test, gconstpointer pdata);
+static void test_ncm_stats_dist_nd_sampling (TestNcmStatsDistNd *test, gconstpointer pdata);
+
```

Comment:
These tests are the ones you talked to me about last time? That when we try to merge something into the library, it tests all the functions inside it? And the x mark in my commits was because we didn't have these tests?